specs/credentialmanagement/index.src.html

```diff
@@ -495,16 +512,17 @@ <h4 id="interfaces-credential-types-credential"><code>Credential</code></h4>
     interface Credential {
       readonly attribute DOMString id;
       readonly attribute DOMString type;
+      boolean isMatch (any filter);
     };
     Credential implements Transferable;
   </pre>
   <dl dfn-for="Credential">
-    <dt><dfn attribute>id</dfn></dt>
+    <dt><dfn attribute for="Credential">id</dfn></dt>
```

Comment:
Nit: You don't need to add the `for` attributes; they're implicitly added by the `dfn-for` attribute on the containing `<dl>`.